python/lsst/ts/electrometer/controller.py

```diff
@@ -297,7 +305,8 @@ async def set_range(self, set_range):
         set_range : `float`
             The new range value.
         """
-        if set_range == -1:
+        self.log.debug(f"type:{type(set_range)} {set_range=}")
```

Comment:
If you really only want set_range to be an integer then it would be better to raise using `isinstance` than to log the value.

If you insist on logging, please add some context, e,g, "type of set_range". Logging the type should not actually be necessary if you log the repr of the value.

Reply:
I was making sure that the comparison was actually between two numbers and therefore would do what I think it should be doing.